### ISSUE TITLE
Used before,  but doesn't work since new update

### ISSUE THREAD

**Issue by eSleepy** [da2x/EdgeDeflector#48]
I used this before, but since the new update it doesn't change anything.  Asks the right questions, but Cortana doesn't really care and uses bing and edge to deliver results.

**Comment by da2x** [comment]
Have you installed the [latest version of EdgeDeflector](https://github.com/da2x/EdgeDeflector/releases/latest)? What is your full Windows version (press Win key+R, type winver, and press Enter)? Is the  “Optimize taskbar web search results for screen readers” option enabled in Microsoft Edge: Settings: View advanced settings?

 > Optimize taskbar web search results for screen readers and i switched it off.

**Comment by eSleepy** [comment]
Yes Sir, I have the latest version installed.
Windows 10 version 1709 OS build 16299.125 
I went and found everything that I could in MS Edge regarding this and turned it off.  I did not find the exact statement that you have here, but I did find one regarding "Have Cortana assist me in MS Edge".  It was enabled and I turned it off.  Results are still the same at this point.  I'll try a reboot as well.

Thanks for your assistance.

**Comment by eSleepy** [comment]
Yes Sir, I have the latest version installed.
Windows 10 version 1709 OS build 16299.125 
I went and found everything that I could in MS Edge regarding this and turned it off.  I did not find the exact statement that you have here, but I did find one regarding "Have Cortana assist me in MS Edge".  It was enabled and I turned it off.  Results are still the same at this point.  I'll try a reboot as well.

Thanks for your assistance.
